src/cmd/build.rs

```diff
@@ -201,7 +201,7 @@ fn exec_cargo_for_wasm_target(
     // Currently will override user defined RUSTFLAGS from .cargo/config. See https://github.com/paritytech/cargo-contract/issues/98.
     std::env::set_var(
         "RUSTFLAGS",
-        "-C link-arg=-z -C link-arg=stack-size=65536 -C link-arg=--import-memory",
+        "-C link-arg=-zstack-size=65536 -C link-arg=--import-memory",
```

Comment:
For some reason (which I don't fully understand) there is an issue with the Windows CI currently failing on `master` if the `-z` is passed separately of the `stack-size` argument, which actually belongs to it. Since both versions should be equal I would change it to this.